apps/dapp/src/hooks/use-geo-blocked.tsx

```diff
@@ -6,10 +6,13 @@ export const useGeoBlocked = () => {
 
   useEffect(() => {
     const checkBlocked = async () => {
-      const blocked = await fetch(`${window.location.href}api/geoblock`)
+      const blocked = await fetch(`${window.location.host}/api/geoblock`)
```

Comment:
We have to manually add the protocol, because this is just the host now, so should be 
```
`https://${window.location.host}/api/geoblock`
```